sdk/rdbms/azure-mgmt-rdbms/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Release History
 
+## 1.10.0 (2020-02-21)
```

Comment:
version should be 2.0.0 as it contains general breaking changes

Reply:
Changed